### PR TITLE
[4.0]namespace administrator components com_modules

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

**Comment by brianteeman** [comment]
now that the system test is actually run can someone explain to me what is failing and how to fix

File: administrator/components/com_modules/View/Modules/HtmlView.php [hunk 1]
```diff
@@ -142,48 +148,48 @@ protected function addToolbar()
...
 		if ($user->authorise('core.create', 'com_modules') && $user->authorise('core.edit', 'com_modules')
 			&& $user->authorise('core.edit.state', 'com_modules'))
 		{
-			\JHtml::_('bootstrap.renderModal', 'collapseModal');
-			$title = \JText::_('JTOOLBAR_BATCH');
+			HTMLHelper::_('bootstrap.renderModal', 'collapseModal');
+			$title = Text::_('JTOOLBAR_BATCH');
 
-			// Instantiate a new \JLayoutFile instance and render the batch button
+			// Instantiate a new \FileLayout instance and render the batch button
```
Comment (Quy) [inline administrator/components/com_modules/View/Modules/HtmlView.php]: Remove \

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**